mne/io/eeglab/eeglab.py

```diff
@@ -7,6 +7,12 @@
 import os.path as op
 
 import numpy as np
+from scipy.io import loadmat
+try:
+    from scipy.io.matlab import MatlabOpaque, MatlabFunction
+except ImportError:  # scipy < 1.8
+    from scipy.io.matlab.mio5_params import MatlabOpaque
+    from scipy.io.matlab.mio5 import MatlabFunction
```

Comment:
these are probably the cause of the non-nested import errors (note: not a circular import, this is a nesting that we insist upon in a test, for certain specific modules that have large load time impacts)

Reply:
this would also be a good candidate for `mne.fixes.py`, a function with API like `MatlabOpaque, MatlabFunction = _import_matlab_classes()`.